errors/abci.go

```diff
@@ -0,0 +1,101 @@
+package errors
+
+import (
+	"errors"
+	"reflect"
+)
+
+// ABCIInfo returns the ABCI error information as consumed by the tenderemint
+// client.
+// This function provides a full error infromation
+func ABCIInfo(err error, debug bool) (uint32, string) {
```

Comment:
Please document return value (code, log)

Reply:
Done in https://github.com/iov-one/weave/pull/399/commits/dbd5b8771d4bef5c83b47d4b198b09fd44a3fda1